src/main/java/pl/put/poznan/buildingInfo/logic/locations/Room.java

```diff
@@ -1,14 +1,26 @@
 package pl.put.poznan.buildingInfo.logic.locations;
 
 import lombok.*;
+import lombok.experimental.SuperBuilder;
 import pl.put.poznan.buildingInfo.logic.visitors.Visitor;
 
 @Getter
 @Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@SuperBuilder
 public class Room extends AbstractLocation {
+    protected Float area;
+    protected Float cube;
+    protected Float lightPower;
 
-    public Room(int id, Float area, Float volume, Float lightPower) {
-        super(id, area, volume, lightPower);
+    protected Float heating;
+    public Room(int id, String name,Float area, Float cube, Float lightPower, Float heating) {
+        super(id, name);
+        this.area = area;
+        this.cube = cube;
+        this.lightPower = lightPower;
+        this.heating = heating;
```

Comment:
jest takie pole w klasie? i czy potrzebujemy zarówno heating jak i lightPower?

Reply:
Jest.  Heating nie robimy niby, ale skoro jest w strukturze pokoju wedlug opisu to w niczym nie szkodzi tutaj.
Po prostu będzie nie wykorzystane.

Reply:
Racja, zapomniałem, że w tym opisie zadania jest też heating